backend/src/main/java/com/javahelp/backend/data/ISurveyResponseStore.java

```diff
@@ -0,0 +1,67 @@
+package com.javahelp.backend.data;
+
+import com.amazonaws.regions.Regions;
+import com.javahelp.model.survey.SurveyResponse;
+import com.javahelp.model.user.User;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+public interface ISurveyResponseStore {
```

Comment:
Agreed. Will be adding an update method.

Reply:
Actually, since SurveyResponse class doesn't have setters for either the survey objects or the responses objects, I don't see the purpose of adding an update method. Rather, the easier way to implement the idea is to always delete prior surveys created by the same user before adding a new survey entry.

What do you think?

Reply:
I personally think your approach of making a new survey each time sounds better, because each survey should have a different `id` with it, even if it was done by the same `User`. 

I feel like the `SurveyResponse` was implemented that way to keep a result version distinct from other survey answers.

Reply:
I agree with this, there does not seem to be much use in an update method